Ahmed-Yazen-CSS/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+    box-sizing: border-box;
+    margin: 0 ;
+    padding: 0;
+    background-color: #F8F8F8;
+}
+main{
+    border: 1px solid gray;
+}
+.header-box {
+   display: inline-block;
+   width: 100%;
+   height: 15%;
+   margin: 0 auto;
+   box-shadow: 2px 2px 10px gray;
+   background-color: #F8F8F8;
+   font-size: large;
+}
+.logo{
+    width:50px;
+    height: 50px;
+    position: relative;
+    left:20%;
```

Comment:
It's not always a good practice to use `position:relative` and use `left` `right` properties with it instead we use `position;absolute` and we give the parent `position:relative` so we can move the node freely.

also try to never use `%` with positioning attributes like `left:20%', `right:20%', ..etc and use `px` instead